index.html

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Please add a doctype declaration this ensures that the web page is parsed the same way by different web browsers.